voice/voice-ai/x/Actions/ActionsView.swift

```diff
@@ -197,21 +199,32 @@ struct ActionsView: View {
     func handleOtherActions(actionType: ActionType) {
         switch actionType {
         case .reset:
-            SpeechRecognition.shared.reset()
+            DeepgramASR.shared.reset()
+            
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
+                DeepgramASR.shared.setup()
+            }
+            break;
         case .skip:
             stopRecording()
```

Comment:
need to update this function - still hooked to Apple's

Reply:
Also I guess skip is not implemented properly in synthesiser anyway - there is no function that is advancing the audio playback by 5 seconds (or whatever is maximally permitted)